### PR TITLE
More cache clarification

### PR DESCRIPTION
**Pull request by plaindocs**
Fix for #416

I think this covers it. @MariadeAnton want to confirm?

### CODE REVIEW

**Comment by MariadeAnton** [comment]
Also, https://github.com/travis-ci/docs-travis-ci-com/issues/357, thank you! 🎉

**Comment by plaindocs** [comment]
Yes, I knew I remembered something about Docker, but was struggling to track it down. The comment was not on a PR but on a commit I think.

File: user/caching.md [hunk 1]
```diff
@@ -206,10 +209,11 @@ If none of the previous locations contain a valid cache, the build continues wit
 
 After the first pull request build is run, it creates a new pull request cache.
 
-Two important things to note about caching for pull requests:
+Some important things to note about caching for pull requests:
 
 * If a repository has *Build pushes* set to *OFF*, neither the target branch nor the master branch can ever be cached.
-* If the cache found is old, for example in a workflow where most work happens on branches, the less useful the cache will be.
+* If the cache on the master branch is old, for example in a workflow where most work happens on branches, the less useful the cache will be.
+* If a pull request has a cache, to clear that cache you need to clear **both** the pull request cache **and** the cache of the target branch.
```
Comment (MariadeAnton) [inline user/caching.md]: maybe we could say "to not use a cache"? 

to clear the PR cache you clear just the pull request cache, but to not fetch any cache, you'd need to clear both, does that make sense?

Comment (plaindocs) [inline user/caching.md]: ah. Yeah, I misunderstood how that worked.

Comment (plaindocs) [inline user/caching.md]: > If a pull request is using a cache but you don't want it to, you need to clear **both** the pull request cache **and** the cache of the target branch.

Comment (MariadeAnton) [inline user/caching.md]: That'd be perfect, thanks @plaindocs!

Comment (plaindocs) [inline user/caching.md]: Cherry picket it into gh-pages.

**Pull request merged**